.github/workflows/velox_docker.yml

```diff
@@ -155,32 +146,39 @@ jobs:
           name: velox-native-lib-${{github.sha}}
           path: ./cpp/build/releases
       - name: Update mirror list
+        if: matrix.os == 'centos:8'
```

Comment:
I combine `run-tpc-test-centos7` and `run-tpc-test-centos8`. It seems the main difference is mirror.